packages/formation-react/src/components/Table/Table.stories.jsx

```diff
@@ -65,32 +86,54 @@ MissingData.args = {
   ...defaultArgs,
   data: [
     {
-      column1: null,
-      column2: 'Things',
+      title: 'A document',
+      description: null,
+      year: null,
     },
     defaultArgs.data[1],
   ],
 };
 
+// This story might not be useful
 export const AlignLeft = Template.bind({});
 AlignLeft.args = {
   ...defaultArgs,
   fields: [
+    defaultArgs.fields[0],
+    defaultArgs.fields[1],
     {
-      label: 'Cost ($)',
-      value: 'cost',
+      label: 'Year',
+      value: 'year',
       alignLeft: true,
     },
-    defaultArgs.fields[1],
   ],
+};
+
+export const CustomComponents = Template.bind({});
+CustomComponents.args = {
+  ...defaultArgs,
   data: [
+    ...defaultArgs.data,
     {
-      cost: 50.5,
-      column2: 'Food',
-    },
-    {
-      cost: 70.0,
-      column2: 'Utilities',
+      title: 'Social Security Act',
+      description: (
+        <>
+          <div>
+            An act to provide for the general welfare by establishing a system
+            of Federal old-age benefits. Enables provisions for:
+          </div>
+
+          <ul>
+            <li>aged persons</li>
+            <li>blind persons</li>
+            <li>dependent and crippled children</li>
+            <li>maternal and child welfare</li>
+            <li>public health</li>
+            <li>unemployment compensation</li>
+          </ul>
```

Comment:
This came from https://www.archives.gov/historical-docs/social-security-act

![image](https://user-images.githubusercontent.com/2008881/98056662-524e7a80-1df5-11eb-92d9-f62c20abba04.png)